src/google/protobuf/stubs/atomicops.h

```diff
@@ -194,14 +194,6 @@ Atomic64 Release_Load(volatile const Atomic64* ptr);
 #elif defined(GOOGLE_PROTOBUF_OS_AIX)
 #include <google/protobuf/stubs/atomicops_internals_power.h>
 
-// Apple.
```

Comment:
Tested with `make clean && ./configure CXXFLAGS="-mmacosx-version-min=10.7" && make` and it doesn't affect the versions the existing code builds at:
\>= 10.9 required with C++11 set
\>= 10.7 required when deleting it from configure.ac